src/wallet/wallet2.cpp

```diff
@@ -2708,6 +2722,9 @@ void wallet2::pull_blocks(uint64_t start_height, uint64_t &blocks_start_height,
   req.start_height = start_height;
   req.no_miner_tx = m_refresh_type == RefreshNoCoinbase;
 
+  req.include_pool_info = try_incremental;
```

Comment:
and this always true?

Reply:
Not a separate issue, connected with other questions